truffle/contracts/Energy.sol

```diff
@@ -6,32 +6,39 @@ contract Energy {
     uint public unitPrice;
     uint public offeredAmount;
     mapping (address => uint) public remainingEnergy;
+    uint public boughtAmount;  // sum of remainingEnergy;
```

Comment:
soldAmount - how much we sold and is left to consume (we offer offered amount and keep track of sold amount)

Reply:
will fix